pkg/schedule/checker/affinity_checker.go

```diff
@@ -282,10 +282,14 @@ func (c *AffinityChecker) createAffinityOperator(region *core.RegionInfo, group
 // - Does NOT skip recently split or recently started regions (as requested)
 // - Only merges regions within the same affinity group
 func (c *AffinityChecker) MergeCheck(region *core.RegionInfo, group *affinity.GroupState) []*operator.Operator {
+	maxSize := int64(c.conf.GetMaxAffinityMergeRegionSize())
```

Comment:
GetMaxMergeRegionSize also need, for example,MaxMergeRegionSize is zero and MaxAffinityMergeRegionSize is not zero, we need a metrics to show